asapdiscovery-alchemy/asapdiscovery/alchemy/cli/prep.py

```diff
@@ -79,6 +79,12 @@ def create(filename: str, core_smarts: str):
     type=click.Path(resolve_path=True, exists=True, file_okay=True, dir_okay=False),
     help="The name of the JSON file which contains the prepared receptor complex including the crystal ligand.",
 )
+@click.option(
```

Comment:
We use `--structure-dir` for this argument in `docking` and `wfs`, may be good to keep consistent.